rotkehlchen/db/upgrades/v39_v40.py

```diff
@@ -334,6 +339,38 @@ def _add_new_tables(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _add_new_tables')
 
 
+def _reset_decoded_events(write_cursor: 'DBCursor') -> None:
+    """
+    Reset all decoded evm events except the customized ones for ethereum mainnet,
+    optimism and polygon.
```

Comment:
Isn't it also for arbitrum by this release? This is copy pasta from v38->v39

Reply:
yes